Services/UserService/Startup.cs

```diff
@@ -2,7 +2,7 @@
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
-
+using UserService.Commands;
 namespace UserService
```

Comment:
add blank line between using block and namespace

Reply:
fixed